docs/Table of Contents.md

```diff
@@ -0,0 +1,62 @@
+# Terraform Provider Documentation
+
+This table of contents shows the resources for files in the /datasources path and the /resources path.
+
+Datasources|  Resources
```

Comment:
In Terraform's documentation they make data source two words - https://www.terraform.io/docs/configuration/data-sources.html.